### ISSUE TITLE
Found 0 crawlable start pages

### ISSUE THREAD

**Issue by Butsch94** [vikrambajaj22/Spidey-Focused-Web-Crawler#2]
Thank you again for the help with the first issue. 
Unfortunately I have another issue.
The Crawler never finds any crawlable start pages.
What could i change? 

C:\Users\schue\Desktop\Webcrawler-A\Spidey-Focused-Web-Crawler-master\venv\Scripts\python.exe C:/Users/schue/Desktop/Webcrawler-A/Spidey-Focused-Web-Crawler-master/spidey.py
Enter your query (default: "wildfires california"):
Enter the number of start pages (default: 10):
Enter the number of pages to be returned (at least 10, default: 1000):
Enter the max. no. of links to be fetched from each page (at least 10, default: 25):
Enter mode 'bfs' or 'focused' (default: 'bfs'):
Enter the relevance threshold (min: 0, max: 4.75, default: 1):

Obtaining start pages...

Found 0 crawlable start pages:

Queue:

No exit code appears even after more than an hour.
So I had to stop the run after 2 hours.

**Comment by vikrambajaj22** [comment]
Hi, are you certain that your computer is connected to the internet and that your IDE isn't blocked by a firewall? I just re-cloned this repository and tried running spidey.py locally, and it ran without any issues.

**Comment by Butsch94** [comment]
hey, 
Thanks for your feedback. I am connected to the internet of course.
I checked my firewall settings and Kaspersky, nothing was blocked. At least I can't see that. 
Do you have any other idea what the issue could be? 
I am currently using PyCharm, should I use another program?

**Comment by vikrambajaj22** [comment]
I would try running from an external terminal, like cmd, Anaconda prompt or Git Bash

**Comment by Butsch94** [comment]
I have tried all of this, however the exact same error message comes up there as well. 
What else could I do so that this could possibly work?

**Comment by vikrambajaj22** [comment]
I am not able to reproduce this error. Please debug the get_start_pages function, as a start.
